trade.lic

```diff
@@ -1,78 +1,419 @@
 =begin
   Documentation: https://elanthipedia.play.net/Lich_script_repository#trade
 =end
-custom_require.call(%w[common common-items common-money common-travel drinfomon equipmanager events])
+custom_require.call(%w[common common-items common-arcana common-crafting common-money common-travel drinfomon equipmanager events])
 
 class Trade
   include DRC
+  include DRCA
   include DRCT
   include DRCI
+  include DRCC
   include DRCM
 
   def initialize
     @town_data = get_data('town')
     @equipment_manager = EquipmentManager.new
     @settings = get_settings
     @contract_container = @settings.trade_contract_container
-    @hometown = @settings.hometown
-    bput("open my #{@contract_container}", 'You open', 'That is already')
-    @equipment_manager.empty_hands
+    @hub_city = 'Crossing'
+    @emergency_delivery_town = nil
+    @current_caravan_destination = nil
+    @caravan_coins_on_hand = @settings.caravan_coins_on_hand.to_i || 25555
```

Comment:
Im not sure that the `25555` will ever be loaded, since this is a base setting. If base doesn't load then you have big problems.

Reply:
Yep, didn't catch that one when I was removing temporary declarations